service/src/main/java/bio/terra/workspace/service/workspace/flight/create/workspace/CreateWorkspaceV2Flight.java

```diff
@@ -57,30 +69,40 @@ public CreateWorkspaceV2Flight(FlightMap inputParameters, Object applicationCont
           new CreateWorkspacePoliciesStep(
               workspace, policyInputs, appContext.getTpsApiDispatch(), userRequest),
           serviceRetryRule);
+
+      addStep(
+          new LinkSpendProfilePolicyAttributesStep(
+              workspace.workspaceId(), workspace.spendProfileId(), appContext.getTpsApiDispatch()),
+          serviceRetryRule);
+
+      // If we're cloning, we need to copy the policies from the source workspace.
```

Comment:
This code was in the original [WorkspaceCreateFlight](https://github.com/DataBiosphere/terra-workspace-manager/blob/47dcbd5918e76349165a16230ec9369cf0de9819/service/src/main/java/bio/terra/workspace/service/workspace/flight/create/workspace/WorkspaceCreateFlight.java#L74) and is needed now that this flight is also used for cloning. There was test coverage that failed when I did not have this in the right place.